src/devices/tuya.ts

```diff
@@ -9864,9 +9902,9 @@ export const definitions: DefinitionWithExtend[] = [
         },
     },
     {
-        fingerprint: tuya.fingerprint("TS0225", ["_TZE200_2aaelwxk"]),
+        fingerprint: [...tuya.fingerprint("TS0225", ["_TZE200_2aaelwxk"]), ...tuya.fingerprint("CK-BL702-MWS-01(7016)", ["_TZE200_2aaelwxk"])],
```

Comment:
```suggestion
        zigbeeModel: ["CK-BL702-MWS-01(7016)"],
        fingerprint: tuya.fingerprint("TS0225", ["_TZE200_2aaelwxk"]),
```

The zigbeeModel seems unique already.